src/test/java/catan/controllers/user/UserControllerTest.java

```diff
@@ -141,4 +155,53 @@ public void shouldLogoutSuccessfullyWithHttpCode200WhenAnyTokenIsPassed() {
                 statusCode(200).
                 body(isEmptyString());
     }
+
+    // end of code which needs to be refactored (see the first line of the file)
+
+
+
+    @Test
+    public void should_successfully_register_and_login_guest() {
+        String userToken = registerAndLoginGuest(USER_NAME_GUEST_1)
+                .then()
+                .statusCode(200)
+                .body("token", not(equalTo(isEmptyString())))
```

Comment:
this is a list of available hamcrest matchers that can be used instead of this shit: not(equalTo(isEmptyString()))